src/test/java/com/ithaca/user/UserTest.java

```diff
@@ -0,0 +1,30 @@
+package com.ithaca.user;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Created by David on 3/24/16.
+ */
+public class UserTest {
+
+    @Test
+    public void testUser() {
+        // Make a user with no construction values
+        User userEmpty = new User();
+        Assert.assertNotNull(userEmpty);
```

Comment:
Does this work? I don't see a constructor for a user without parameters? Am I missing something?